tasks/release.js

```diff
@@ -120,7 +120,7 @@ module.exports = function(gulp, plugins, _, config) {
 		function(done) {
 			var branchName = 'bower-staging-' + Math.random().toString(16).replace(/[^0-9a-fA-F]/g, '');
 			var browserCommitMessage = 'Browser files for v' + bumpedVersion;
-			var mergeCommitMessage = 'Merging ' + BRANCH_RELEASE + '@v' + bumpedVersion + ' into ' + BRANCH_DEV;
+			var mergeCommitMessage = 'Merging ' + BRANCH_RELEASE + '@v' + bumpedVersion + ' into ' + BRANCH_STABLE;
```

Comment:
Funning thing is, I don't think this merge commit has _ever_ gotten created. If I look at the history of _every_ commit on every branch (21k lines) with `git log --all --grep 'into 1.x-develop'` I don't find any evidence that we've ever actually done this (ie. zero hits, including when you look at where 1.x-develop used to point before I deleted it, which is 3039054de).

My guess is that the merge fails for some reason and we just silently swallow the error.